nbhttp/processor.go

```diff
@@ -37,7 +37,6 @@ func releaseRequest(req *http.Request) {
 		if req.Body != nil {
 			br := req.Body.(*BodyReader)
 			br.close()
-			bodyReaderPool.Put(br)
```

Comment:
where are you freeing this now?

Reply:
here:
https://github.com/lesismal/nbio/blob/multi_allocator/nbhttp/body.go#L83